test/e2e/unifiedpush_test.go

```diff
@@ -0,0 +1,57 @@
+package e2e
+
+import (
+	"testing"
+	"time"
+
+	apis "github.com/aerogear/unifiedpush-operator/pkg/apis"
+	aerogearv1alpha1 "github.com/aerogear/unifiedpush-operator/pkg/apis/aerogear/v1alpha1"
+	framework "github.com/operator-framework/operator-sdk/pkg/test"
+	"github.com/operator-framework/operator-sdk/pkg/test/e2eutil"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+var (
+	retryInterval        = time.Second * 5
+	timeout              = time.Second * 200
+	cleanupRetryInterval = time.Second * 1
+	cleanupTimeout       = time.Second * 5
+)
+
+func TestUnifiedpush(t *testing.T) {
+	unifiedpushList := &aerogearv1alpha1.UnifiedPushServerList{
+		TypeMeta: metav1.TypeMeta{
```

Comment:
I think you can remove this `TypeMeta` section. It should be implied by the fact that this is an `&aerogearv1alpha1.UnifiedPushServerList`, and having it set explicitly here could lead to weirdness when we get to `v1beta1` or other versions: where we might get compiler/tooling help to update the versions of objects, these string properties could be missed.